libcxx/test/std/time/time.point/time.point.arithmetic/op_++.pass.cpp

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: std-at-least-c++20
+
+// <chrono>
+
+// time_point
+
+// constexpr time_point& operator++();
+
+#include <cassert>
+#include <chrono>
+
+#include "test_macros.h"
+
+constexpr bool test() {
+  typedef std::chrono::system_clock Clock;
+  typedef std::chrono::milliseconds Duration;
+  std::chrono::time_point<Clock, Duration> t{Duration{5}};
+  std::chrono::time_point<Clock, Duration>& tref{++t};
+  return &tref == &t && tref.time_since_epoch() == Duration{6};
+}
+
+int main(int, char**) {
+  assert(test());
```

Comment:
```suggestion
  std::chrono::time_point<Clock, Duration> t{Duration{5}};
  std::chrono::time_point<Clock, Duration>& tref{++t};
  assert(&tref == &t);
  assert(tref.time_since_epoch() == Duration{6});
  
  return true;
}

int main(int, char**) {
  test();
```
I probably didn't explain it quite clear. You want to write the `assert()` and the test function as in the example above. This makes it much clearer what failed when you have multiple `asserts()`.